submissions/oluwaboriShobayo/CSS/productPage.css

```diff
@@ -0,0 +1,237 @@
+/* This section is for the media screen query*/
```

Comment:
This task is for contact us page. Why are we have styling for product page? Kindly remove it